Vagrantfile

```diff
@@ -18,6 +18,7 @@ KUBEADM_INIT_FLAGS = ENV["KUBEADM_INIT_FLAGS"] || ''.freeze
 if KUBERNETES_VERSION != "" && KUBEADM_INIT_FLAGS == ""
     KUBEADM_INIT_FLAGS = "--kubernetes-version=#{KUBERNETES_VERSION}"
 end
+KUBE_NETWORK = ENV['KUBE_NETWORK'] || 'flannel'.freeze
```

Comment:
is this default value of `flannel` going to prevent the statement from the readme from being correct?
> What CNI to install, if empty don't install any CNI.

if the user explicitly calls make with `KUBE_NETWORK=` then will their desire to not install a CNI be fulfilled?